test/Microsoft.Framework.DependencyInjection.Tests/Utils/MultiServiceHelpers.cs

```diff
@@ -25,6 +25,11 @@ public static IEnumerable GetMultiService(Type collectionType, Func<Type, IEnume
 
         private static IEnumerable Cast([NotNull] IEnumerable collection, Type castItemsTo)
         {
+            if (collection == null)
```

Comment:
I see, this was removed by your PR: 277d338643f439992892275126b49bf1d6e2579f
but I made this PR before yours was merged.